hack/jenkins/upload-finished.sh

```diff
@@ -29,9 +29,17 @@ if [[ $# -ne 1 ]]; then
     exit 1
 fi
 
+# TODO: DRY. Refactor into upload-to-gcs.sh ?
+: ${JENKINS_GCS_LOGS_PATH:="gs://kubernetes-jenkins/logs"}
+: ${JENKINS_UPLOAD_TO_GCS:="y"}
+
+if [[ ! ${JENKINS_UPLOAD_TO_GCS:-} =~ ^[yY]$ ]]; then
```

Comment:
`if [[ ! "${JENKINS_UPLOAD_TO_GCS:=y}" =~ ^[yY]$ ]]; then`

That way you don't need to set it a few lines above. Not a big deal, though.